.pre-commit-config.yaml

```diff
@@ -2,35 +2,36 @@ repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
     rev: "v0.2.1"
     hooks:
-        - id: ruff
```

Comment:
`pre-commit` was using 4 space and 2 space tabs, this just makes it 2 space tabs.